Workbooks/Windows Virtual Desktop/AtScale/User Report/User Report.workbook

```diff
@@ -1334,7 +1334,7 @@
                   "type": 3,
                   "content": {
                     "version": "KqlItem/1.0",
-                    "query": "WVDConnections\r\n| where UserName == '{UserName}'\r\n| as SourceData\r\n| where State==\"Started\"\r\n| project-rename StartTime=TimeGenerated\r\n| join kind=fullouter\r\n(\r\n    SourceData\r\n    | where State == \"Completed\"\r\n    | project CorrelationId, SessionHostName, EndTime=TimeGenerated, _ResourceId\r\n) on CorrelationId\r\n| extend  HostPool=extract_all(\"(?i:resourceGroups/([^/]+)/providers/microsoft.desktopvirtualization/hostpools/([^/]+))\",coalesce(_ResourceId, _ResourceId1))[0]\r\n| project CorrelationId=coalesce(CorrelationId, CorrelationId1), StartTime=coalesce(StartTime, {TimeRange:start}), EndTime=coalesce(EndTime, {TimeRange:end}), HostPool=strcat(HostPool[0],\"-\",HostPool[1])\r\n| extend ProbeTime = range(bin_at(StartTime, {TimeRange:grain}, {TimeRange:start}), bin_at(EndTime + {TimeRange:grain}, {TimeRange:grain}, {TimeRange:end}), {TimeRange:grain})\r\n| mv-expand ProbeTime to typeof(datetime)\r\n| extend ConnectionContainsProbe = (StartTime < ProbeTime and EndTime > ProbeTime),\r\n         ConnectionEndedInPreviousBucket = (StartTime < ProbeTime and EndTime + {TimeRange:grain} >= ProbeTime) \r\n| make-series Connections=dcountif(CorrelationId, ConnectionContainsProbe or ConnectionEndedInPreviousBucket) on ProbeTime from {TimeRange:start} to {TimeRange:end} step {TimeRange:grain} by HostPool",
+                    "query": "WVDConnections\r\n| where UserName == '{UserName}'\r\n| as SourceData\r\n| where State==\"Started\"\r\n| summarize arg_min(TimeGenerated, SessionHostName, _ResourceId) by CorrelationId\r\n| project-rename StartTime=TimeGenerated\r\n| join kind=fullouter\r\n(\r\n    SourceData\r\n    | where State == \"Completed\"\r\n    | summarize arg_max(TimeGenerated, SessionHostName, _ResourceId) by CorrelationId\r\n    | project-rename EndTime=TimeGenerated) on CorrelationId\r\n| extend  HostPool=extract_all(\"(?i:resourceGroups/([^/]+)/providers/microsoft.desktopvirtualization/hostpools/([^/]+))\",coalesce(_ResourceId, _ResourceId1))[0]\r\n| project CorrelationId=coalesce(CorrelationId, CorrelationId1), StartTime=coalesce(StartTime, {TimeRange:start}), EndTime=coalesce(EndTime, {TimeRange:end}), HostPool=strcat(HostPool[0],\"-\",HostPool[1])\r\n| extend ProbeTime = range(bin_at(StartTime, {TimeRange:grain}, {TimeRange:start}), bin_at(EndTime + {TimeRange:grain}, {TimeRange:grain}, {TimeRange:end}), {TimeRange:grain})\r\n| mv-expand ProbeTime to typeof(datetime)\r\n| extend ConnectionContainsProbe = (StartTime < ProbeTime and EndTime > ProbeTime),\r\n         ConnectionEndedInPreviousBucket = (StartTime < ProbeTime and EndTime + {TimeRange:grain} >= ProbeTime) \r\n| make-series Connections=dcountif(CorrelationId, ConnectionContainsProbe or ConnectionEndedInPreviousBucket) on ProbeTime from {TimeRange:start} to {TimeRange:end} step {TimeRange:grain} by HostPool",
```

Comment:
Nit: seems like the `) on CorrelationId` part of the join has moved up into the previous line, and that doesn't follow the style guide used in the rest of the workbooks